drivers/mpu9250/mpu9250.c

```diff
@@ -52,89 +68,223 @@ typedef struct
   uint16_t mag_x;
   uint16_t mag_y;
   uint16_t mag_z;
-} __attribute__((packed)) buffer_t;
+} __attribute__((packed)) polling_data_t;
+
+typedef struct
+{
+  uint16_t buf_acc_x[1024];
+  uint16_t buf_acc_y[1024];
+  uint16_t buf_acc_z[1024];
+  uint16_t buf_gyro_x[1024];
+  uint16_t buf_gyro_y[1024];
+  uint16_t buf_gyro_z[1024];
+} __attribute__((packed)) buffer_data_t;
```

Comment:
`acc` and `gyro` values are fine, but we should at least add a timestamp array too. any thoughts?

Reply:
Just added a single timestamp, which is used to mark the buffer start. This should be sufficient, since all following values have a 1ms spacing and thus can be calculated.

Furthermore, the way the timestamping is currently implemented in HW, isn't consistent anyway, from what I can see.  However, I would say we can ignore that for now. Let's focus on the main functionality first. :slightly_smiling_face:

Reply:
Well, I dont know if that makes it any better. If I have to calculate the timestamps myself I could simply use software timestamps as well..

Since the driver has nothing to do with timestamp generation at all the HW should fix its inconsistencies :P

(and yes, functionality has top prio, but without proper timestamping the displayed values in grafana are inaccurate)